chrome/content/zotero/elements/notesContext.js

```diff
@@ -277,8 +278,7 @@
 				vbox.style.display = 'flex';
 
 				editor = new (customElements.get('note-editor'));
-				editor.style.display = 'flex';
-				editor.style.width = '100%';
+				editor.setAttribute("flex", "1");
```

Comment:
Why is this necessary? Seems like we want to be moving away from the `flex` attribute.

Reply:
in 115 the `flex="1"` equals to `style="flex: 1"`. but if we have agreed to remove `flex`, then yes we can use the style.